fullcalendar-app/src/components/FullCalendar.vue

```diff
@@ -215,23 +250,34 @@ export default {
         clone.querySelectorAll('.fc-listDay-view tbody td')
           .forEach(function(element) {
             element.style.height = '3rem';
-        });
+          });
       }
-
-      clone.querySelectorAll('.fullcalendar--header, .fc-header-toolbar .fc-toolbar-chunk:not(:first-child), .calendar-branch-info h4')
-        .forEach(function(element) {
+    },
+    cleanupUnwantedElements(clone) {
+      const elementsToHide = clone.querySelectorAll(
+        '.fullcalendar--header, .fc-header-toolbar .fc-toolbar-chunk:not(:first-child), .calendar-branch-info h4'
+      );
+      elementsToHide.forEach(element => {
         element.style.display = 'none';
       });
-
-      // Adding copy to the DOM for PDF generation. The copy will be deleted immediately, so the user will not see the change.
+    },
+    addCloneToDOM(clone) {
       document.body.appendChild(clone);
+    },
+    generatePDF(clone) {
+      const scheduleTitle = document.querySelector('.fc-toolbar-title').textContent;
+      const filename = `Weekly schedule (${scheduleTitle}).pdf`;
```

Comment:
Can we remove white spaces from the file name, please?

Reply:
Fixed.